src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -136,7 +136,7 @@ public static NetWorth parseNetWorth(String netWorth) throws ParseException {
      */
     public static MeetingTime parseMeetingTime(String meetingTime) throws ParseException {
         requireNonNull(meetingTime);
-        String trimmedMeetingTime = meetingTime.trim();
+        String trimmedMeetingTime = meetingTime.replaceAll("\\s", "");
```

Comment:
What is the point of using this replaceAll?
Does this trim the spaces?

Reply:
Yup effectively removes all whitespace
Using trim or strip only removes leading and trailing whitespaces I think, doesnt remove the inbetween ones